### ISSUE TITLE
Download photos form icloud

### ISSUE THREAD

**Issue by liwenjie0728** [horrorho/InflatableDonkey#84]
if there is a method to Download photos form icloud?

**Comment by 0x311fc486c1** [comment]
No, not with this tool although it should be possible to extend this code and provide the feature. Other option would be to try to use [pyicloud](https://github.com/picklepete/pyicloud) and see if that still works - or you could still use it at least as an inspiration.

**Comment by herrgutt** [comment]
Yes there is, @liwenjie0728 . Sign in to iCloud for your PC, then select to download all photos/videos.
